css/jquery.imageplayer.css

```diff
@@ -40,11 +40,11 @@
 }
 
 /*
```

Comment:
what is the reason to take this out?

Reply:
Are you talking about the lines of

/*
img_player .stage img {
    image-rendering: optimizeQuality;
    -ms-interpolation-mode: bicubic;
}
*/

This is commented out in your last commit on your master branch.